src/pages/resources/asset-compute-worker-ps-api/lesson3.md

```diff
@@ -67,23 +67,27 @@ exports.main = worker(async (source, rendition, params) => {
 });
 ```
 
-Because the SDK requires `apiKey` to be passed in the input params, update your manifest to include it.
+Because the SDK requires `apiKey` to be passed in the input params, update your `app.config.yaml` to include it.
```

Comment:
For asset compute, the manifest info now moves to ext.config.yaml inside the extension folder, not app.